tests/PintaBenchmarks/Utilities/Utilities.cs

```diff
@@ -0,0 +1,19 @@
+using Pinta.Core;
+
+namespace PintaBenchmarks;
+
+internal static class Utilities
+{
+	private static readonly IServiceManager mock_services;
+
+	static Utilities ()
+	{
+		mock_services = CreateMockServices ();
+	}
+
+	private static IServiceManager CreateMockServices ()
```

Comment:
I haven't tried running the benchmarks with these changes, but will it cause any runtime errors if there isn't a palette provided now?

Reply:
No, it shouldn't cause any runtime errors. This was added because the constructors of those effects now receive an `IServiceManager`, but I was careful so that the default behaviors are the exact same ones as before the changes.

Reply:
I get the following from running `dotnet run --project tests/PintaBenchmarks -c Release -- -f \*JuliaFractalEffect`

```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.ApplicationException: Could not resolve service type Pinta.Core.IPaletteService
   at Pinta.Core.ServiceManager.GetService[T]() in /Users/cameron/code/Pinta/Pinta.Core/Managers/ServiceManager.cs:line 55
   at Pinta.Effects.JuliaFractalEffect..ctor(IServiceManager services) in /Users/cameron/code/Pinta/Pinta.Effects/Effects/JuliaFractalEffect.cs:line 35
   at PintaBenchmarks.EffectsBenchmarks.JuliaFractalEffect() in /Users/cameron/code/Pinta/tests/PintaBenchmarks/EffectsBenchmarks.cs:line 96
```

Reply:
You are right...The mock palette should be created and added to the service manager first. I just added it; this error you got shouldn't happen again